src/components/StorageOverview/Inventory/diskInventoryUtils.js

```diff
@@ -0,0 +1,24 @@
+import { get } from 'lodash';
+
+import { getCapacityStats } from '../../../selectors';
+import { STATUS_RESULT_OK, STATUS_RESULT_ERROR } from '../../Dashboard/Inventory/utils';
+
+const result = {
```

Comment:
please put this inside the function. There is no reason to have it outside

Reply:
Ack!